src/main/java/seedu/duke/Duke.java

```diff
@@ -3,19 +3,20 @@
 import java.util.Scanner;
 
 public class Duke {
-    /**
-     * Main entry-point for the java.duke.Duke application.
-     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("What is your name?");
 
+        System.out.println("Hello! I'm FridgeFriend!");
+        System.out.println("What can I do for you?\n");
+
+        String input;
         Scanner in = new Scanner(System.in);
-        System.out.println("Hello " + in.nextLine());
+        input = in.nextLine();
+        while (!input.equals("bye")){
```

Comment:
Would it be better to use a switch statement here? This can help accommodate future addition of commands.

Reply:
Alright